docs/source/timing-analysis.rst

```diff
@@ -474,36 +523,40 @@ The ``diff_ecdf_plot.png`` is the ECDF counterpart to the scatter plot.
 Here, if the graph is
 `symmetrical
 <https://en.wikipedia.org/wiki/Symmetric_probability_distribution>`_ then the
-results from the Wilcoxon signed-rank test are meaningful. If the graph
-is asymmetric focus on sign test results.
+results from the Wilcoxon signed-rank will be robust. If the graph
+is asymmetric, sign test results should be more robust.
 The ``diff_ecdf_plot_zoom_in_98.png``, ``diff_ecdf_plot_zoom_in_33.png``,
 and ``diff_ecdf_plot_zoom_in_10.png`` show just the central 98, 33, and 10
 percentiles respectively of the graph (to make estimating small differences
 between samples easier).
 
 Finally, the ``conf_interval_plot_mean.png``,
 ``conf_interval_plot_median.png``, ``conf_interval_plot_trim_mean_05.png``,
-``conf_interval_plot_trim_mean_25.png``, and ``conf_interval_plot_trimean.png``
-show the mean, median, trimmed mean (5%), trimmed mean (25%), and trimean
+``conf_interval_plot_trim_mean_25.png``,
+``conf_interval_plot_trim_mean_45.png``, and ``conf_interval_plot_trimean.png``
+show the mean, median, trimmed mean (5%), trimmed mean (25%), trimmed mean
+(45%), and trimean
 respecively, of the differences between samples together with
 `bootstrapped
 <https://en.wikipedia.org/wiki/Bootstrapping_(statistics)>`_ confidence
 interval for them.
 For an implementation without a timing side channel present, all the graphs
 should intersect with the horizonal 0 line.
-If a graph does not intersect with the 0 line, then the number of heights
-of it from the 0 line suggests how strong is the confidence in the
-presence of side channel on an exponential scale.
+If a graph does not intersect with the 0 line, then the distance of it
+from the 0 line suggests how strong is the confidence in the
+presence of side channel is on an exponential scale.
```

Comment:
This whole sentence is difficult to read. Can this be split or rewritten so it is more understandable? Or maybe some commas would help.

Reply:
rewritten, please check

Reply:
looks good